jetstream/outcomes/firefox_desktop/relay.toml

```
@@ -0,0 +1,120 @@
+friendly_name = "Relay"
+description = "Metrics for the Relay integration"
+
+
+# Metric for tracking how many times the offer was shown
+[metrics.relay_integration_shown_relay_offer]
+friendly_name = "Relay offer shown to client"
+description = "Number of times Relay offer is shown"
+select_expression = """
+      COALESCE(COUNTIF(
+         event_method= 'shown_offer_relay'
+         AND event_category = 'relay_integration'
+      ),0)
```

change/suggestions (blocking): If these outcome queries pull from the same `events` source as the sql.telemetry.mozilla.org queries' `telemetry.events`, then this query needs to change to:
```suggestion
      COALESCE(COUNTIF(
         event_method = 'shown'
         AND event_object = 'offer_relay'
         AND event_category = 'relay_integration'
      ),0)
```

You can see the existing sql.telemetry.mozilla.org relay integration opt-in conversion funnel query here: https://sql.telemetry.mozilla.org/queries/93179/source#230436